src/domain/FlowRunner.ts

```diff
@@ -545,8 +551,13 @@ export class FlowRunner implements IFlowRunner, IFlowNavigator, IPromptBuilder {
     )
 
     interactions.push(richCursor.interaction)
+
+    // step cursor forward with context
```

Comment:
Is this comment necessary? Is so, how could it be made into code? (as comments become stale, and indicate that clarity/naming needs to be better, or that a piece of code needs to be extracted into its own function)